src/horizon/payment_call_builder.ts

```diff
@@ -5,13 +5,19 @@ import { ServerApi } from "./server_api";
  * Creates a new {@link PaymentCallBuilder} pointed to server defined by serverUrl.
  *
  * Do not create this object directly, use {@link Server#payments}.
- * @see [All Payments](https://developers.stellar.org/api/resources/payments/)
+ * @see [All Payments](https://developers.stellar.org/api/horizon/resources/list-all-payments/)
  * @constructor
  * @extends CallBuilder
  * @param {string} serverUrl Horizon server URL.
  */
 export class PaymentCallBuilder extends CallBuilder<
-  ServerApi.CollectionPage<ServerApi.PaymentOperationRecord>
+  ServerApi.CollectionPage<
+    | ServerApi.PaymentOperationRecord
+    | ServerApi.CreateAccountOperationRecord
+    | ServerApi.AccountMergeOperationRecord
+    | ServerApi.PathPaymentOperationRecord
+    | ServerApi.PathPaymentStrictSendOperationRecord
```

Comment:
when soroban is released it will be possible for invoke host operations to be included in the payments response. Can you also include invoke host operations in this list?

Reply:
great point, thank you! done